__tests__/articles.test.js

```diff
@@ -27,7 +27,8 @@ describe("GET /api/articles/:article_id", () => {
                 body: 'I find this existence challenging',
                 created_at: '2020-07-09T20:11:00.000Z',
                 votes: 100,
-                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                comment_count: 11
```

Comment:
Should be a separate test rather than a refactor of the previous on.  Please make this previous test less brittle, possibly make use of jest matcher expect.ObjectContaining, to just check the presence of properties rather than testing for strict equality to a fixed object.  Then please add in a new test checking for the presence of comment_count.